examples/rt685s-evk/src/lib.rs

```diff
@@ -13,7 +18,85 @@ static OTFAD: [u8; 256] = [0; 256];
 #[rustfmt::skip]
 #[link_section = ".fcb"]
 #[used]
-static FCB: FlexSPIFlashConfigurationBlock = FlexSPIFlashConfigurationBlock::build();
+static FCB_685EVK: FlexSPIFlashConfigurationBlock = FlexSPIFlashConfigurationBlock::build().lookup_table([
```

Comment:
odd, the default configuration should already be working on the EVK. Does it not work for you?

Reply:
The default one does not seem to work anymore, the upgrade 0.1 to 0.2 changed the default: https://github.com/OpenDevicePartnership/embassy-imxrt/pull/385#discussion_r2037554525

Reply:
approving this for now, as it will prevent regressions on the FCB from breaking the examples.